src/plugins/intel_gpu/src/plugin/ops/unary.cpp

```diff
@@ -85,7 +85,7 @@ static void CreatePReluOp(Program& p, const std::shared_ptr<ngraph::op::v0::PRel
         if (!ngraph::op::util::get_single_value(slope_node, slope))
             IE_THROW() << "Unsupported parameter size in " << op->get_friendly_name() << " (" << op->get_type_name() << ")";
         CreateUnaryEltwiseOp(p, op, cldnn::activation_func::relu_negative_slope, {slope});
-    } else if (out_shape.size() >= 2 && ngraph::shape_size(slope_shape) == out_shape[1]) {
```

Comment:
Could you explain why the second condition removed?

Reply:
For example, in the case where the shape of the slope tensor is [1, N, M, 1], `ngraph::shape_size(slope_shape)` is `N x M` and `out_shape[1]` is `N`. So I remove the second condition.